src/osemosys/schemas/model.py

```diff
@@ -59,7 +60,7 @@ def to_simplicity(self, comparison_directory) -> Dict[str,str]:
         Dict[str,str]
             A dictionary with keys the otool filenames and paths the otool paths
         """
-        self.commodities.to_simplicity(comparison_directory)
+        self.time_definition.to_csv(comparison_directory, self.time_definition)
```

Comment:
hmmm okay. we probably want `to_csv[simplicity/otool/whatever]` to be just a normal instance method, so we don't also need to pass an instance of 'TimeDefinition'. `from_csv[simplicity/otool]` should be a classmethod though.